docs/functionality-overview.md

```diff
@@ -67,7 +67,10 @@ Only a subset of HTML is converted to Govspeak.
 
 Elements that are converted, but with attributes stripped are:
 
-p, h2, h3, blockquote, pre, code, ol, ul, li, br
+p, h1, h2, h3, h4, h5, blockquote, pre, code, ol, ul, li, br
+
+Note: h1 headers are converted to h2 headers, h4 and h5 headers are converted
```

Comment:
I'd suggest a small reordering here:
Note: As GOV.UK only encourages the use of h2 and h3 headers, h1 headers are converted to h2 headers, and h4 and h5 headers are converted to h3 headers.